VAMobile/src/utils/rnAuthSesson.tsx

```diff
@@ -19,3 +19,11 @@ export const startAuthSession = async (codeChallenge: string): Promise<string> =
     return await CustomTabs.beginAuthSession(AUTH_SIS_ENDPOINT, codeChallenge)
   }
 }
+
+export const clearCookies = async (): Promise<boolean> => {
+  if (isIOS()) {
+    return await RnAuthSession.clearCookies()
```

Comment:
unfortunately not. there is no way for us to unit test this in react native that I know of aside from the function being called, which we know it is due to me having to add the mock.

natively we may be able to test it, however it would require a lot of setup I believe for our CLI to run those unit tests. I will create a spike ticket to add unit tests for native modules